### PR TITLE
Replaced the material parameters of ThermoHydroMechanics with MPL

### PR DESCRIPTION
**Pull request by wenqing**
as titled.

1. [x] Feature description was added to the [changelog](https://github.com/ufz/ogs/wiki/Release-notes-6.2.2)
2. [x] Tests covering your feature were added? With the existing THM bechmarks.
3. [ ] Any new feature or behavior change was documented? Not needed.

### CODE REVIEW

File: ProcessLib/ThermoHydroMechanics/ThermoHydroMechanicsFEM-impl.h [hunk 1]
```diff
@@ -310,20 +352,46 @@ void ThermoHydroMechanicsLocalAssembler<ShapeFunctionDisplacement,
...
+
+        MTT.noalias() +=
+            N_T.transpose() * effective_volumetric_heat_capacity * N_T * w;
 
         //
         // temperature equation, pressure part
         //
-        KTp.noalias() += K_over_mu * rho_f * C_f * N_T.transpose() *
-                         (dNdx_T * T).transpose() * dNdx_T * w;
+        KTp.noalias() += fluid_density * c_f * N_T.transpose() *
```
Comment (TomFischer) [inline ProcessLib/ThermoHydroMechanics/ThermoHydroMechanicsFEM-impl.h]: Just out of curiosity: Why did you reorder the calculation?

File: ProcessLib/ThermoHydroMechanics/ThermoHydroMechanicsFEM-impl.h [hunk 2]
```diff
@@ -215,51 +225,76 @@ void ThermoHydroMechanicsLocalAssembler<ShapeFunctionDisplacement,
...
-        auto const rho_sr = _process_data.solid_density(t, x_position)[0];
-        auto const rho_fr = _process_data.fluid_density(t, x_position)[0];
-        auto const porosity = _process_data.porosity(t, x_position)[0];
+
         auto const& b = _process_data.specific_body_force;
         auto const& identity2 = MathLib::KelvinVector::Invariants<
             MathLib::KelvinVector::KelvinVectorDimensions<
                 DisplacementDim>::value>::identity2;
 
+         //TODO: Change dT to time step wise increment
```
Comment (endJunction) [inline ProcessLib/ThermoHydroMechanics/ThermoHydroMechanicsFEM-impl.h]:
```suggestion
         // TODO (wenqing): Change dT to time step wise increment
```

Comment (wenqing) [reply ProcessLib/ThermoHydroMechanics/ThermoHydroMechanicsFEM-impl.h]: Added.

**Comment by wenqing** [comment]
@TomFischer  I did the changes manually.

File: ProcessLib/ThermoHydroMechanics/ThermoHydroMechanicsFEM-impl.h (hunk 1 above)
Comment (wenqing) [reply ProcessLib/ThermoHydroMechanics/ThermoHydroMechanicsFEM-impl.h]: Because K_over_mu is changed to matrix now.

**Pull request merged**